Makefile.build.mk

```diff
@@ -0,0 +1,49 @@
+export GOPROXY=https://proxy.golang.org
+export CGO_ENABLED=0
+
+SHELL := /bin/bash -o pipefail
+VERSION_PACKAGE = github.com/replicatedhq/kots/pkg/buildversion
```

Comment:
this still references kots, but it also seems like we may be able to get rid of this and some other stuff not used in the main Makefile